gitindex/index_test.go

```diff
@@ -384,15 +384,64 @@ func TestIndexDeltaBasic(t *testing.T) {
 					addedDocuments: branchToDocumentMap{
 						"main": []zoekt.Document{sourcegraphIgnoreWithContent},
 					},
-					optFn: func(t *testing.T, options *Options) {
-						options.BuildOptions.IsDelta = true
+					optFn: func(t *testing.T, o *Options) {
+						o.BuildOptions.IsDelta = true
 					},
 
 					expectedFallbackToNormalBuild: true,
 					expectedDocuments:             []zoekt.Document{sourcegraphIgnoreWithContent},
 				},
 			},
 		},
+		{
+			name:     "should fallback to a full, normal build if the repository has more than the specified threshold of shards",
```

Comment:
This test will be thrown away after we implement a better strategy, but I thought it was worth adding to verify the  behavior of this simple one.